devenv/README.md

```diff
@@ -2,11 +2,43 @@
 
 A Developer Environment (or devenv for short) is a workspace for developers to make changes without breaking anything in a live environment. 
 
-In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it'own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors. 
+In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it's own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors.
 
-# Instructions 
+We have included all the dependencies that you need to build the Netdata Agent and it's collectors. Moreover, we have included helpful tools such as `python pip` and `flake8`. 
 
-- Clone this repository 
-- `cd` into the *devenv/docker* directory
-- run `docker build -t netdata_devenv:latest` and `docker run -p 19999:19999 devenv`
-  
\ No newline at end of file
+You only need to download your forked repositories and you can begin developing right away!
+
+# Dockerfile
+
+Inside this repository you can find the Dockerfile which is used to create the Docker Container. It is automatically published on [DockerHub](https://hub.docker.com/r/netdata/devenv), using a CI/CD pipeline based on GitHub Actions. 
```

Comment:
```suggestion
Inside this repository you can find the Dockerfile which is used to create the development environment Docker image. It is automatically published on [DockerHub](https://hub.docker.com/r/netdata/devenv), using a CI/CD pipeline based on GitHub Actions. 
```

Without a change like this, the wording is somewhat ambiguous, and could be interpreted to mean that this is the Dockerfile used for the official Netdata Agent images.